render/mapping.go

```diff
@@ -48,10 +48,14 @@ func MapEndpointIdentity(m RenderableNode, local report.Networks) RenderableNode
 	}
 
 	// We need to filter out short lived connections from this view,
-	// if they don't also have a pid; see #447
+	// if they don't also have a pid; see #447.  Note if they
+	// we're introduced by proc spy then they are guaranteed to
+	// have a pid on the other end of the adjacency, so we include them
+	// no matter what.
 	pid, pidOK := m.Metadata[process.PID]
 	_, conntracked := m.Metadata[endpoint.Conntracked]
-	if !pidOK && conntracked {
+	_, procspied := m.Metadata[endpoint.Procspied]
+	if !procspied && !pidOK && conntracked {
```

Comment:
Note this could now probably just be !procspied, as no conntracked nodes have a pid, they were only getting pids through merges with procspied nodes.  Should I simplify?